locales.json

```diff
@@ -24,244 +13,290 @@
-      "imprintAndContact": "Imprint and contact",
-      "privacy": "Data Privacy",
-      "offers": "Offers",
-      "categories": "Categories",
-      "events": "Events",
-      "noEvents": "No events",
-      "noLanguages": "Not available in other languages",
-      "noTranslation": "No translation available",
-      "search": "Search",
-      "changeLocation": "Change location",
-      "changeLanguage": "Change language"
+      "createPdf": "Create a PDF"
+    },
+    "es": {
+      "createPdf": "Crear PDF"
     },
     "fa": {
-      "imprintAndContact": "مجوز ‎و اطلاعات تماس",
-      "privacy": "محافظت از داده ها",
-      "categories": "دسته​ها",
-      "noLanguages": "در زبان های دیگر موجود نیست",
-      "noTranslation": "ترجمه موجود نیست",
-      "search": "جستجو",
-      "changeLocation": "تغییر محل",
-      "changeLanguage": "تغییر زبان"
+      "createPdf": "ایجاد پی دی اف"
     },
     "fr": {
-      "imprintAndContact": "Mentions légales et contact",
-      "privacy": "Protection des données",
-      "categories": "Catégorie",
-      "noLanguages": "Non disponible dans d'autres langues",
-      "noTranslation": "Aucune traduction disponible",
-      "search": "Rechercher",
-      "changeLocation": "Modifier le lieu",
-      "changeLanguage": "Changer la langue"
-    }
-  },
-  "search": {
-    "de": {
-      "searchCategory": "Suche nach einer Kategorie"
+      "createPdf": "Créer PDF"
     },
-    "en": {
-      "searchCategory": "Search for a category"
-    }
-  },
-  "categories": {
-    "de": {
-      "createPdf": "PDF erstellen"
+    "hr": {
+      "createPdf": "Kreiraj PDF"
     },
-    "en": {
-      "createPdf": "Create PDF"
+    "ku": {
+      "createPdf": "Pêkanîna PDF'ê"
     },
-    "fr": {
-      "createPdf": "Créer PDF"
+    "pl": {
+      "createPdf": "Wygeneruj PDF"
     },
-    "ar": {
-      "createPdf": "إنشاء ملف PDF"
+    "ro": {
+      "createPdf": "Creați PDF"
     },
-    "fa": {
-      "createPdf": "پی دی اف بسازید"
+    "ru": {
+      "createPdf": "Создать PDF"
+    },
+    "so": {
+      "createPdf": "Abuur PDF"
+    },
+    "ti": {
+      "createPdf": "PDF ፍጠር"
+    },
+    "tr": {
+      "createPdf": "PDF oluşturun"
     }
   },
   "common": {
-    "en": {
-      "languageNotAvailable": "Sorry, but your language is not available here.",
-      "chooseALanguage": "Choose a language.",
-      "lastUpdate": "Last Update: "
-    },
     "de": {
       "languageNotAvailable": "Deine Sprache ist hier leider nicht verfügbar.",
       "chooseALanguage": "Wähle eine Sprache.",
       "lastUpdate": "Letzte Änderung: "
     },
-    "fr": {
-      "languageNotAvailable": "Malheureusement, ta langue n'est pas disponible ici.",
-      "chooseALanguage": "Choisissez votre Langue."
-    },
     "ar": {
       "languageNotAvailable": "للأسف، لغتك ليست متاحة.",
-      "chooseALanguage": "اختار اللغة.",
-      "lastUpdate": "اخر تحديث: "
+      "chooseALanguage": "اختر لغة.",
+      "lastUpdate": "التغيير الأخير:"
```

Comment:
~~Is the punctuation right here? I thought since it's RTL the colon should be on the left side?~~
Okay, just an issue of github. Displayed correctly in intelliJ